keyboards/cannonkeys/atlas_alps/config.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2020 Nathan Spears
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xCA04
+#define PRODUCT_ID      0xA7A5
+#define DEVICE_VER      0x0010
+#define MANUFACTURER    CannonKeys
```

Comment:
???

Reply:
The keyboard itself was manufactured by Cannon Keys. It's my design. Or am I missing something?

Reply:
Okay.  In that case, it's a bit muddy, but should be okay. 

Just in case, @awkannan.